File: js/src/0.0-Basics/0.3-build-up-logical-thinking/0.3.14-pattern.js
function log(val) {
    process.stdout.write(String(val));
}

function patter_14() {
    let row = 5;
    let column = 5;

    let counter = 1;

    for (let y = 1; y < row + 1; y++) {
        for (let x = 0; x < y; x++) {
            log(`${counter} `);
            counter = counter + 1;
        }
        console.log();
    }
}

patter_14();

File: python/0.0-Basics/0.3-build-up-logical-thinking/0.3.14-pattern.py
from utils.clear import clear


def pattern_14():
    row = 5
    column = 5

    counter = 1

    for i in range(1, row + 1):
        for j in range(1, column + 1):
            if j <= i:
                print(counter, " ", end="")
                counter = counter + 1
            elif j > i:
                print(" ", end="")

        print()


clear()
pattern_14()
